src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/BlobEncoders.cs

```diff
@@ -149,7 +205,20 @@ public MethodSignatureEncoder(BlobBuilder builder, bool hasVarArgs)
             HasVarArgs = hasVarArgs;
         }
 
-        public void Parameters(int parameterCount, out ReturnTypeEncoder returnType, out ParametersEncoder parameters)
+        /// <summary>
+        /// Encodes return type and parameters.
+        /// 
+        /// Returns a pair of encoders that must be used in the order they appear in the tuple:
+        /// first <see cref="ReturnTypeEncoder"/> to encode the return type,
+        /// then <see cref="ParametersEncoder"/> to encode the actual parameters.
+        /// </summary>
+        /// <param name="parameterCount">Number of parameters.</param>
+        /// <returns>
+        /// Returns a pair of encoders that must be used in the order they appear in the tuple:
+        /// first <see cref="ReturnTypeEncoder"/> to encode the return type,
+        /// then <see cref="ParametersEncoder"/> to encode the actual parameters.
+        /// </returns>
```

Comment:
Do we need to duplicate this text? From offline discussion, the issue is that returns docs don't show up in IntelliSense and this is important to read. I would just strip the repetitive returns doc.